scripts/agent_app.py

```diff
@@ -0,0 +1,100 @@
+"""
```

Comment:
The idea is similar to the one in the EVO repository, but here, I used the DeployableAgent class instead of benchmark ones.